### PR TITLE
release: v0.1.2 with fixes to migrate and export 

### PR DESCRIPTION
**Pull request by rhamzeh**

### CODE REVIEW

File: lib/helpers/openfga/fga.adapter.ts [hunk 1]
```diff
@@ -52,6 +52,10 @@ export interface OpenFgaClientConfig {
   immutableBypassSecret?: string;
 }
 
+const AUTH0_PROD_HOSTS = [Auth0FgaConfiguration.getEnvironmentConfiguration(KnownEnvironment.US).apiHost,
```
Comment (adriantam) [inline lib/helpers/openfga/fga.adapter.ts]: should these constant be in this file ?  Or will it be better to move it to a separate auth0fga specific file and import from it?

Comment (adriantam) [reply lib/helpers/openfga/fga.adapter.ts]: Or if this file is meant for auth0fga specifically, maybe rename it as such?

**Review by adriantam — DISMISSED** [review]
Those are just comments and can be done in a separate PR

File: lib/helpers/openfga/fga.adapter.ts (hunk 1 above)
Comment (rhamzeh) [reply lib/helpers/openfga/fga.adapter.ts]: These are just env specific configs, moved them to their own file

**Review by adriantam — APPROVED** [review]

**Pull request merged**